responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/household-members/household-members.component.ts

```diff
@@ -53,12 +54,16 @@ export class HouseholdMembersComponent implements OnInit, OnDestroy {
     // Main form creation
     this.createHouseholdForm();
 
+    // Primary Member must always be in Mebmbers array
+    this.members = [this.stepEssFileService.primaryMember];
```

Comment:
@stairaku @MShayOffice I think this is needed only for Create New ESS File and not for other scenarios. Can you confirm?